common/component_interface_utils/include/component_interface_utils/status.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef COMPONENT_INTERFACE_UTILS__STATUS_HPP_
+#define COMPONENT_INTERFACE_UTILS__STATUS_HPP_
+
+namespace component_interface_utils::status
+{
+
+template <class T1, class T2>
+void copy(const T1 & src, T2 & dst)  // NOLINT cpplint false positive
```

Comment:
Could you write the error message here?

Reply:
Is it like this https://github.com/autowarefoundation/autoware.universe/pull/1921/commits/1c5b53ba94bfe272c67df8c7798f8ee7690e7da3? The error message is:
```
Add #include <algorithm> for copy  [build/include_what_you_use] [4]
```

Reply:
In addition, this also causes false positives in execution.
```
  component_interface_utils::status::copy(a, b);  // cpplint requires algorithm header
```

Reply:
Ah, no. I just wanted to know this one. Thanks!

```
Add #include <algorithm> for copy  [build/include_what_you_use] [4]
```

Reply:
> In addition, this also causes false positives in execution.

Maybe related? https://github.com/cpplint/cpplint/issues/200